scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/definition/component/parameter/defaults/SimpleLanguageDeterminer.scala

```diff
@@ -0,0 +1,36 @@
+package pl.touk.nussknacker.engine.definition.component.parameter.defaults
+
+import pl.touk.nussknacker.engine.api.definition._
+import pl.touk.nussknacker.engine.graph.expression.Expression
+
+trait SimpleLanguageDeterminer {
```

Comment:
I think this would be better as an `object`, not a `trait`, so essentially "composition over inheritance", as @arkadius says

also `EditorBasedLanguageDeterminer` would be a more telling name IMO

Reply:
fixed